lib/session/inbound-call-session.js

```diff
@@ -33,13 +33,24 @@ class InboundCallSession extends CallSession {
   }
 
   _onCancel() {
+    this.rootSpan.setAttributes({'call.termination': 'caller abandoned'});
     this.rootSpan.setAttributes({'call.termination': 'caller abandoned'});
     this.callInfo.callTerminationBy = 'caller';
+    const wasEarlyMedia = this.callInfo.callStatus === 'early-media';
     this._notifyCallStatusChange({
       callStatus: CallStatus.NoAnswer,
       sipStatus: 487,
       sipReason: 'Request Terminated'
     });
+    if (wasEarlyMedia) {
```

Comment:
I don't think we need this event. Please consider to remove it.

NoAnswer is already good enoug as it's final status as same as Completed